workshops/personal_website/examples/versions.md

```diff
@@ -0,0 +1,162 @@
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+
+</body>
+</html>
+```
+
+--------------------------------------------------------------------------------
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  I am Drake
+  All your hotlines belong to me.
+</body>
+</html>
+```
+
+--------------------------------------------------------------------------------
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <h1>I am Drake</h1>
+  All your hotlines belong to me.
+</body>
+</html>
+```
+
+--------------------------------------------------------------------------------
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+--------------------------------------------------------------------------------
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <img src="https://media.giphy.com/media/vsyKKf1t22nmw/giphy.gif">
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+--------------------------------------------------------------------------------
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <img src="https://media.giphy.com/media/vsyKKf1t22nmw/giphy.gif">
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+--------------------------------------------------------------------------------
+
+```css
+img {
+    width: 200px;
+}
+```
+
+--------------------------------------------------------------------------------
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+}
+```
+
+--------------------------------------------------------------------------------
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+    background-color: #34495e;
+}
+```
+
+--------------------------------------------------------------------------------
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+    background-color: #34495e;
+    color: white;
+}
+```
+
+--------------------------------------------------------------------------------
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+    background-color: #34495e;
+    color: white;
+    font-family: Arial;
+}
+```
+
+--------------------------------------------------------------------------------
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <title>I am Drake</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <img src="https://media.giphy.com/media/vsyKKf1t22nmw/giphy.gif">
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
```

Comment:
Is this file just for your reference? Or is this going to be in the workshop as well?